install.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+
+
+[ ! -d ~/.tldr ] && printf "tldr seems to have already been previously installed.\nDelete the \'$HOME/.tldr\' directory first before relaunching this script\n" && exit 1 || mkdir -p ~/.tldr/cache
```

Comment:
Indeed it's not very needed, I've deleted this in a further commit (commit [bb3a00658f74f49c3271621e467193b91c2a9241](https://github.com/j8r/tldr/commit/bb3a00658f74f49c3271621e467193b91c2a9241))